common/src/gas_provider/node.rs

```diff
@@ -70,6 +71,32 @@ impl<T> From<ReservationId> for GasNodeId<T, ReservationId> {
     }
 }
 
+pub type NodeLock<Balance> = [Balance; 4];
```

Comment:
impl Zero maybe?

Reply:
obvious thing. newtype pattern will help

Reply:
This idea has also come across my mind. Just didn't want to complicate things unnecessarily - that's why tried to get away with the `Default` instead. Anyway, I agree that implementing `Zero` for the lock type (whatever it is) would be useful